sdk/keyvault/azure-keyvault-administration/CHANGELOG.md

```diff
@@ -32,6 +46,11 @@
   - `denied_actions` -> `not_actions`
   - `allowed_data_actions` -> `data_actions`
   - `denied_data_actions` -> `denied_data_actions`
+- Renamed argument `role_assignment_name` to `name` in
+  `KeyVaultAccessControlClient.create_role_assignment`, `.delete_role_assignment`,
+  and `.get_role_assignment`
+- Renamed argument `role_definition_name` to `name` in
+  `KeyVaultAccessControlClient.delete_role_definition` and `.get_role_definition`
 
```

Comment:
We'll want to mention other argument renames too, like `role_scope` -> `scope` in multiple methods